sql/core/src/main/scala/org/apache/spark/sql/execution/aggregate/utils.scala

```diff
@@ -70,7 +70,6 @@ object Utils {
     // Check if we can use TungstenAggregate.
     val usesTungstenAggregate =
       child.sqlContext.conf.unsafeEnabled &&
-      aggregateExpressions.forall(_.aggregateFunction.isInstanceOf[DeclarativeAggregate]) &&
```

Comment:
This also reminds me: are there planner tests that should be updated to assert that we're actually using the new Tungsten path? Let me check and see.